rpcserver.go

```diff
@@ -4521,16 +4521,55 @@ func (r *rpcServer) FeeReport(ctx context.Context,
 // 0.000001, or 0.0001%.
 const minFeeRate = 1e-6
 
+// checkMaxHtlc errors if the user tries to update their channel policy with
+// a max HTLC value that is too high or too low for a given list of channels.
+func checkMaxHtlc(maxHtlc uint64, channels []*channeldb.OpenChannel) error {
+	var maxHtlcRequested = lnwire.MilliSatoshi(maxHtlc)
+	for _, ch := range channels {
+		if maxHtlcRequested < ch.LocalChanCfg.MinHTLC {
+			return fmt.Errorf("max htlc size of %v msat is below "+
+				"min HTLC policy of %v msat for channel %v",
+				maxHtlc, ch.LocalChanCfg.MinHTLC,
+				ch.ShortChannelID)
```

Comment:
Perhaps use the channel point instead here and below as that's what is input by the user?